sbi/utils/metrics.py

```diff
@@ -1,6 +1,67 @@
 import numpy as np
 import torch
 
+from typing import Optional
+from sklearn.model_selection import KFold, cross_val_score
+from sklearn.neural_network import MLPClassifier
+
+
+def c2st(
+    X: torch.Tensor,
+    Y: torch.Tensor,
+    seed: int = 1,
+    n_folds: int = 5,
+    scoring: str = "accuracy",
+    z_score: bool = True,
+    noise_scale: Optional[float] = None,
+) -> torch.Tensor:
+    """Classifier-based 2-sample test returning accuracy
```

Comment:
Fullstop and new line

Reply:
How about: 
`Returns accuracy of classifier trained to distinguish samples from two distributions.`